x-pack/plugins/ml/public/application/trained_models/models_management/pipelines/expended_row.tsx

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC } from 'react';
+import { EuiBadge, EuiInMemoryTable } from '@elastic/eui';
+import { EuiBasicTableColumn } from '@elastic/eui/src/components/basic_table/basic_table';
+import { i18n } from '@kbn/i18n';
+import { useFieldFormatter } from '../../../contexts/kibana/use_field_formatter';
+import { FIELD_FORMAT_IDS } from '../../../../../../../../src/plugins/field_formats/common';
+import { IngestStatsResponse } from './pipelines';
+
+interface ProcessorsStatsProps {
+  stats: Exclude<IngestStatsResponse, undefined>['pipelines'][string]['processors'];
+}
+
+type ProcessorStatsItem = ProcessorsStatsProps['stats'][number][string] & { id: string };
+
+export const ProcessorsStats: FC<ProcessorsStatsProps> = ({ stats }) => {
+  const durationFormatter = useFieldFormatter(FIELD_FORMAT_IDS.DURATION);
+
+  const items: ProcessorStatsItem[] = stats.map((v, i) => {
+    const key = Object.keys(v)[0];
+    return {
+      ...v[key],
+      id: `${key}_${i}`,
+    };
+  });
+
+  const columns: Array<EuiBasicTableColumn<ProcessorStatsItem>> = [
+    {
+      field: 'type',
+      name: i18n.translate(
+        'xpack.ml.trainedModels.modelsList.pipelines.processorStats.typeHeader',
+        {
+          defaultMessage: 'Processor type',
+        }
+      ),
+      width: '100px',
+      sortable: true,
+      truncateText: false,
+      render: (type: string) => {
+        return <EuiBadge color="hollow">{type}</EuiBadge>;
+      },
+      'data-test-subj': 'mlProcessorStatsType',
+    },
+    {
```

Comment:
Can info icons be added with help popovers for the Count, Time per doc, Current and Failed columns, using the text suggested in the feedback doc?

![image](https://user-images.githubusercontent.com/7405507/141967279-08c8f194-5865-4f2b-839a-9f81b6ecc6aa.png)

Reply:
![image](https://user-images.githubusercontent.com/5236598/141990065-1c85c12e-5b40-4605-b41e-c966b6d60804.png)
Added in 4a5e4bd87e121fccb5e68fbf5a50c3c4f7b91a7d